### PR TITLE
Spacing atom defined

### PR DESCRIPTION
**Pull request by pallavikadam-srijan**
Defined 13 spacing formats in UNDP’s design system, so they can be standardized across UNDP’s sites for various components.

### CODE REVIEW

**Comment by Jura** [comment]
Attaching build log for your perusal
[design-system.2aef7bae-e636-4e1f-8ce0-255f54f22b37.log](https://github.com/undp/design-system/files/7206660/design-system.2aef7bae-e636-4e1f-8ce0-255f54f22b37.log)

File: stories/Atom/Global/Spacing/spacing.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+`}</style>
+
+# Spacing Component
+Space units create visual rhythm and balance.
+
+<ul>
+  <li>Margin is a css property for spacing outside of a component.</li>
+  <li>Padding is a css property for spacing inside of a component.</li>
+  <li>We can use <code>spacing</code> in any component to sets the margin and padding area on all four sides of an element at once.</li>
+</ul>
```
Comment (surbhi90) [inline stories/Atom/Global/Spacing/spacing.stories.mdx:29]: @pallavikadam-srijan Above text seems to be irrelevant to me
CC- @srijan-mehul-gada

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comment

File: stories/assets/scss/_variables.scss [hunk 2]
```diff
@@ -1,2 +1,19 @@
...
+$spacing-04 : 12px;
+$spacing-05 : 16px;
+$spacing-06 : 24px;
+$spacing-07 : 32px;
+$spacing-08 : 40px;
+$spacing-09 : 48px;
+$spacing-10 : 64px;
+$spacing-11 : 80px;
+$spacing-12 : 96px;
+$spacing-13 : 160px;
```
[the remark this answers is not in the document]
Comment (pallavikadam-srijan) [reply stories/assets/scss/_variables.scss]: Done

File: stories/Atom/Global/Spacing/spacing.scss [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    .spacing__title.Primary {
+        font-style: normal;
+        font-weight: bold;
+        font-size: 55px;
+        line-height: 109%; 
+        margin-top: 7px;
+        margin-bottom: 60px;
+    }
+    .spacing__define .spacing__name {
+        display: grid;
```
[the remark this answers is not in the document]
Comment (pallavikadam-srijan) [reply stories/Atom/Global/Spacing/spacing.scss]: Done

File: stories/Atom/Global/Spacing/spacing.stories.mdx (hunk 1 above)
Comment (pallavikadam-srijan) [reply stories/Atom/Global/Spacing/spacing.stories.mdx:29]: @surbhi90 could you please suggest any other relevant text which I can replace here or should I remove this text completely?
